README.md

```diff
@@ -51,10 +51,42 @@ Then, in your `.eslintrc.json`:
 }
 ```
 
+## Recommended Configuration
+
+This plugin works best if you disable all other ESLint rules relating to code formatting, and only enable rules that detect patterns in the AST. (If another active ESLint rule disagrees with `prettier` about how code should be formatted, it will be impossible to avoid lint errors.) You can use [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) to disable all formatting-related ESLint rules.
+
+To integrate this plugin with `eslint-config-prettier`, you can use the `"recommended"` configuration.
+
+Install `eslint-config-prettier` as well:
+
+```sh
+npm install --save-dev eslint-config-prettier
+```
+
+Then all you need in your `.eslintrc.json` is:
+
+```json
+{
+  "extends": [
+    "plugin:prettier/recommended"
+  ]
+}
+```
+
+This does three things:
+
+1. Enables `eslint-plugin-prettier`.
+2. Sets the `prettier/prettier` rule to `"error"`.
+3. Extends the `eslint-config-prettier` configuration.
+
+You can then set Prettier's own options inside a `.prettierrc` file.
+
 ## Options
 
+> Note: While it is possible to pass options to Prettier via your ESLint configuration file, it is not recommended because editor extensions such as `prettier-atom` and `prettier-vscode` **will** read [`.prettierrc`](https://prettier.io/docs/en/configuration.html), but **won't** read settings from ESLint, which can lead to an inconsistent experience.
```

Comment:
As a sidenote: I was under the impression that people using `eslint-plugin-prettier` would just use editor integrations for `eslint`, because it wouldn't be necessary to use integrations for `prettier`. Is it common for people to use `prettier-atom` along with `eslint-plugin-prettier`?

Reply:
I personally use `prettier-vscode` with `eslint-plugin-prettier`. I don't use use the `vscode-eslint` plugin's format on save capabilities as Prettier covers a wider range of languages, and I find it a bit faster.